shared-bindings/wifi/Radio.c

```diff
@@ -163,6 +163,86 @@ STATIC mp_obj_t wifi_radio_connect(size_t n_args, const mp_obj_t *pos_args, mp_m
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_KW(wifi_radio_connect_obj, 1, wifi_radio_connect);
 
+//|     ap_rssi: int
+//|     """RSSI of the currently connected AP. Returns none if not connected"""
+//|
+STATIC mp_obj_t wifi_radio_get_ap_rssi(mp_obj_t self) {
+    return common_hal_wifi_radio_get_ap_rssi(self);
+
+}
+MP_DEFINE_CONST_FUN_OBJ_1(wifi_radio_get_ap_rssi_obj, wifi_radio_get_ap_rssi);
+
+const mp_obj_property_t wifi_radio_ap_rssi_obj = {
+    .base.type = &mp_type_property,
+    .proxy = { (mp_obj_t)&wifi_radio_get_ap_rssi_obj,
+               (mp_obj_t)&mp_const_none_obj,
+               (mp_obj_t)&mp_const_none_obj },
+};
+
+//|     ap_ssid: int
```

Comment:
What do you think about re-using the Network object to represent the current network? I think it has all of the `ap_` attributes you want here. Then you can just return None or a Network object.

Reply:
I think you're right, I'll try that.  That'd avoid pulling out everything individually and would probably be nicer to interact with too.

Reply:
The current network is usually represented as an ip_interface isn't it? See https://docs.python.org/3/library/ipaddress.html#ipaddress.ip_interface 
The network object has the host bits and'ed away.